src/renderer/views/Channel/Channel.js

```diff
@@ -1620,6 +1644,108 @@ export default defineComponent({
+
+      try {
+        /**
+         * @type {import('youtubei.js').YT.Channel}
+         */
+        const channel = this.channelInstance
+        const coursesTab = await channel.getCourses()
+
+        if (expectedId !== this.id) {
+          return
+        }
+
+        this.latestCourses = coursesTab.playlists.map(playlist => parseLocalListPlaylist(playlist, this.id, this.channelName))
+        this.coursesContinuationData = coursesTab.has_continuation ? coursesTab : null
+        this.isElementListLoading = false
+      } catch (err) {
+        console.error(err)
+        const errorMessage = this.$t('Local API Error (Click to copy)')
+        showToast(`${errorMessage}: ${err}`, 10000, () => {
+          copyToClipboard(err)
+        })
+        if (this.backendPreference === 'local' && this.backendFallback) {
+          showToast(this.$t('Falling back to Invidious API'))
+          this.channelInvidiousCourses()
+        } else {
+          this.isLoading = false
+        }
+      }
+    },
+
+    getChannelCoursesLocalMore: async function () {
+      try {
+        /**
+         * @type {import('youtubei.js').YT.ChannelListContinuation}
+         */
+        const continuation = await this.coursesContinuationData.getContinuation()
+
+        const parsedCourses = continuation.playlists.map(playlist => parseLocalListPlaylist(playlist, this.id, this.channelName))
+        this.latestCourses = this.latestCourses.concat(parsedCourses)
+        this.coursesContinuationData = continuation.has_continuation ? continuation : null
+      } catch (err) {
+        console.error(err)
+        const errorMessage = this.$t('Local API Error (Click to copy)')
+        showToast(`${errorMessage}: ${err}`, 10000, () => {
+          copyToClipboard(err)
+        })
+      }
+    },
+
+    channelInvidiousCourses: function() {
+      this.isElementListLoading = true
+
+      getInvidiousChannelCourses(this.id).then((response) => {
+        this.coursesContinuationData = response.continuation || null
+        this.latestCourses = response.playlists
+        this.isElementListLoading = false
+      }).catch(async (err) => {
+        console.error(err)
+        const errorMessage = this.$t('Invidious API Error (Click to copy)')
+        showToast(`${errorMessage}: ${err}`, 10000, () => {
+          copyToClipboard(err)
+        })
+        if (process.env.SUPPORTS_LOCAL_API && this.backendPreference === 'invidious' && this.backendFallback) {
+          showToast(this.$t('Falling back to Local API'))
+          if (!this.channelInstance) {
+            this.channelInstance = await getLocalChannel(this.id)
+          }
+          this.getChannelCoursesLocal()
+        } else {
+          this.isLoading = false
+        }
+      })
+    },
+
+    channelInvidiousCoursesMore: function () {
+      if (this.coursesContinuationData === null) {
+        console.warn('There are no more courses available for this channel')
+        return
+      }
```

Comment:
Is there any situation where we can actually run into this? Looking at the existing logic we hide the load more button when there is no continuation data...

Reply:
Doesn't look like it, I removed the console.warn here (and elsewhere in the file)